deepchecks/nlp/checks/data_integrity/text_property_outliers.py

```diff
@@ -81,10 +81,17 @@ def run_logic(self, context: Context, dataset_kind: DatasetKind) -> CheckResult:
         for name, values in properties.items():
             # If the property is single value per sample, then wrap the values in list in order to work on fixed
```

Comment:
make sure this comment is next to the relevant place + add comment explaining your newly added code

Reply:
Done